src/js/disableUIOnBusy.js

```diff
@@ -2,9 +2,17 @@ function shouldShow() {
     return $("html").hasClass("shiny-busy");
 }
 
+function showModal(){
```

Comment:
Show our generic 'shiny is busy' modal only if the shiny notification panel isn't on the page